frontend/index.html

```diff
@@ -2,7 +2,13 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Pulldasher</title>
+    <title></title>
+    <script type="module" src="./src/page-context.ts"></script>
```

Comment:
I think this probably won't work and breaks out of the webpack bundle, pulling in unbuilt pieces. I'd vote for just leaving this blank or using the technique mentioned in [this SO post](https://stackoverflow.com/a/48750048/13216)